specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/compute.json

```diff
@@ -9462,17 +9474,424 @@
       ],
       "description": "Describes a network interface reference."
     },
+    "VirtualMachineIpTag": {
+      "properties": {
+        "ipTagType": {
+          "type": "string",
+          "description": "IP tag type. Example: FirstPartyUsage."
+        },
+        "tag": {
+          "type": "string",
+          "description": "IP tag associated with the public IP. Example: SQL, Storage etc."
+        }
+      },
+      "description": "Contains the IP tag associated with the public IP address."
+    },
+    "VirtualMachinePublicIPAddressDnsSettingsConfiguration": {
+      "properties": {
+        "domainNameLabel": {
+          "type": "string",
+          "description": "The Domain name label. The concatenation of the domain name label and vm [what] will be the domain name labels of the PublicIPAddress resources that will be created"
+        }
+      },
+      "required": [
+        "domainNameLabel"
+      ],
+      "description": "Describes a virtual machines network configuration's DNS settings."
```

Comment:
We should maybe indicate this is the domain name label prefix.

Reply:
should we rename the field then? and how should we describe given that it is concatenated with the profile configuration ID?

Reply:
this is the name that is used in VMSS already, we want that compatibility.

Although there is different behavior.

We are also using "name" to mean the prefix as well (and in VMSS) so I think this is OK.